simulator/src/constants.py

```diff
@@ -0,0 +1,2 @@
+SCHEMA_PATH= '\schema.py'
```

Comment:
Missing white-space before equal to sign !
Let's also skip the use of any kind of 'slash' in file name to make it platform independent, as \ acts differently on windowns and linux

Also, we can just address it as:
SCHEMA_FILE = 'schema.py'